src/pages/HomePage/hooks.ts

```diff
@@ -39,7 +39,7 @@ const useContact: UseContact = () => {
   const pipelines = useQueryWithClient([QueryKey.PIPELINES], getPipelinesService);
 
   const opportunities = useQueryWithClient(
-    [QueryKey.OPPORTUNITIES, contactId],
+    [QueryKey.OPPORTUNITIES, `${contactId}`],
```

Comment:
Why is `contactId` wrapped it a template?

Reply:
because `react-query` expects a string, but the `contactId` is a number